### DISCUSSION TITLE
`getNodes()` and `getEdges()` are going to be deprecated?

### DISCUSSION THREAD

**Discussion by harshaktg** [bcakmakoglu/vue-flow#1520] [Q&A] [answered]
I see in the file that these methods will be deprecated in the next major version. I'm planning to change the occurrences of these in the code base. I'm unable to find the suggested `useVisibleNodes` and `useVisibleEdges`. Is this something WIP? Or am I missing something here?

<img width="644" alt="image" src="https://github.com/bcakmakoglu/vue-flow/assets/40317596/0e9d7f51-5799-4226-b838-776748efcaee">

**Comment by bcakmakoglu** [comment, marked as answer]
It's a WIP, you can ignore the message for now ^^

Also `getNodes` is *not a function*. It is currently a `computed` so the usage would not be `getNodes()` it would be `getNodes.value`

**Reply by bcakmakoglu** [reply]
FYI: the deprecation notice is removed in `1.38.4` until the mentioned composables actually become available.
